feat: Pull local json

Jeff-Tian opened the pull request:
Thanks for the great package, it helped me much!

Would be possible to merge this improvement for pulling local json file?

If already downloaded the json file, can use the following:

```
npx -p @stackbit/stackbit-pull stackbit-pull-local-json --json-file=/path/to/json
```

I've already used it successfully.

smnh commented:
Hi @Jeff-Tian ,

The stackbit-pull API endpoint receives the `dataFormat: 'object'` property which can be used to pull everything as a single json object. Please see if that solves your issue. If not, lets discuss what is missing in addition to `dataFormat: 'object'`.

```js
const stackbitPull = require('@stackbit/stackbit-pull');

stackbitPull.pull({
  stackbitPullApiUrl: '...',
  apiKey: '...',
  dataFormat: 'object', // return everything as an objects rather converting it plain files by extension
  ver: 2, // ver 2 sets wrap: true by default
  wrap: true, // wrap response in object with projectId and data properties rather returning an array
  metadata: true, // return __metadata field with each object
  resolveLinks: false // resolve links of referenced object, if false, every referenced object will be { stackbit_ref_id: refId }
});
```

smnh reviewed (changes requested):
- in stackbit, we use 4 spaces for indentation
- there are a lot of reformat changes which makes it difficult to review. I suggest rolling them back (with squash) so the diff will include only the changed logic

Jeff-Tian commented:
Hi @smnh , thanks for your quick response!

The change is not about the date format. Rather, the intention is to get the object from local json file. It's super useful when you have the json file already downloaded.

For the format changes, I reverted most of them. Now it should be more clearer about what has been changed:
- Added a stackbit-pull-local-json.js file
- extracted the callback in `then` in stackbit-pull.js into the function named `writeFiles` and reused it in the `createFromLocalJsonFile` function.

smnh commented:
@Jeff-Tian added our standard .prettier config from our CLI, please reformat your PR once again using it.
Thanks.

Jeff-Tian commented:
@smnh done.

smnh commented:
@rodikh is this merged to the `independent` branch you worked on?
Can I merge this one to master and then you update your work from it?

rodikh commented:
Sure, merge this to master and I'll update the feature branch.
